.mdlrc

```diff
@@ -2,4 +2,4 @@
 verbose false
 
 # DISABLE rules for MDL to use on .md files
-rules "~MD013", "~MD005", "~MD009"
+rules "~MD013", "~MD005", "~MD009", "~MD024"
```

Comment:
You may want to just set the `siblings_only` setting for [MD024](https://github.com/DavidAnson/markdownlint/blob/main/doc/Rules.md#md024---multiple-headings-with-the-same-content) as it will still find duplicates where there shouldn't be.

Reply:
that only works in the npm one I think. Maybe we should look at switching to that, thoughts @cdwilhelm ?

Reply:
Swapped it out for npm.